1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,16 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+function getBudgets(peopleArray) {
+  let sumOfBudgets = 0;
+  if (peopleArray === []) {
```

Comment:
It isn't necessary to check for an empty array here because `.forEach()` will still work on an empty array.

This solution is fine, though!